onos-classic/templates/statefulset.yaml

```diff
@@ -48,7 +51,7 @@ spec:
       {{- end }}
       initContainers:
         - name: {{ .Chart.Name }}-init
-          image: tutum/dnsutils:latest
+          image: "{{ .Values.initContainer.repository }}:{{ .Values.initContainer.tag }}"
```

Comment:
Just curious - what is the benefit of parameterizing this? This doesn't seem used in other places.

Reply:
in our environment pod has no access to internet so we have to override all docker images with our internal clones